manifests/server/grant_role.pp

```diff
@@ -0,0 +1,43 @@
+# Define for granting membership to a role. See README.md for more information
+define postgresql::server::grant_role (
+  $group,
+  $role,
+  $ensure           = 'present',
+  $psql_db          = $postgresql::server::default_database,
+  $psql_user        = $postgresql::server::user,
+  $port             = $postgresql::server::port,
+  $connect_settings = $postgresql::server::default_connect_settings,
+) {
+  case $ensure {
+    'present': {
+      $command = "GRANT '${group}' TO '${role}'"
```

Comment:
Is it just the correction of the single quotes to double quotes (identified by @janfabry), and the validation of $group and $role that this PR is pending on? This diff (untested, but pretty straightforward) might clean up those two remaining issues.

```
--- grant_role.pp.orig 	2016-08-30 19:15:53.000000000 -0700
+++ grant_role.pp      	2016-08-30 19:19:54.000000000 -0700
@@ -8,13 +8,15 @@
   $port             = $postgresql::server::port,
   $connect_settings = $postgresql::server::default_connect_settings,
 ) {
+  validate_string($group)
+  validate_string($role)
   case $ensure {
     'present': {
-      $command = "GRANT '${group}' TO '${role}'"
+      $command = "GRANT \"${group}\" TO \"${role}\""
       $unless_comp = '='
     }
     'absent': {
-      $command = "REVOKE '${group}' FROM '${role}'"
+      $command = "REVOKE \"${group}\" FROM \"${role}\""
       $unless_comp = '!='
     }
     default: {
```